### PR TITLE
feat: token price design changes

### PR DESCRIPTION
**Pull request by joshuanatanielnm**
#583 

## Description

update the UI based on the first design iteration

## Changes

List any technical changes.

- [x] create cellar data home map
- [x] create hooks to get token price and 1week change data
- [x] refactor cellarstats component to handle 2 different data
- [x] fix visual bug with incorrect alignment
- [x] create percentage text component to handle text with icon percentage
- [x] update strategy hero data
- [x] comment out 1w change because of business decision 

## Screenshots (if appropriate):

image

image

image

image

## Links

Figma
[Add links to Figma files, documentation, etc.](https://www.figma.com/file/7IAQbLyzapnrow9sIcO77P/App?node-id=2195:101054)

### CODE REVIEW

File: src/components/CellarStats.tsx [hunk 1]
```diff
@@ -14,34 +14,35 @@ import { CurrentDeposits } from "./CurrentDeposits"
...
 import { debounce } from "lodash"
-import { Apy } from "./Apy"
 import { isCurrentDepositsEnabled } from "data/uiConfig"
 import { ConfigProps } from "data/types"
 
 interface CellarStatsProps extends StackProps {
-  tvm?: ReactNode
-  apy?: ReactNode
-  apyTooltip?: string
+  firstTooltip?: string
```
Comment (codingki) [inline src/components/CellarStats.tsx:21]: Can we have a better naming instead using "first" and "second"

File: src/components/CellarStats.tsx [hunk 2]
```diff
@@ -67,45 +67,44 @@ export const CellarStats: VFC<CellarStatsProps> = ({
...
-          }, 1000)}
-        >
-          <Tooltip
-            hasArrow
-            placement="top"
-            label={overrideApy?.tooltip || apyTooltip}
-            bg="surface.bg"
-            color="neutral.300"
+      {/* REMOVE THIS CONDITION IF WE WANT TO DISPLAY 1W CHANGE PERCENTAGE */}
+      {isAave && (
```
Comment (codingki) [inline src/components/CellarStats.tsx:85]: put this in `src/data/uiConfig.ts` as `isTokenPriceWeeklyChangeEnabled`

File: src/components/_cards/CellarCard/ValueManaged.tsx [hunk 3]
```diff
@@ -26,21 +30,61 @@ export const ValueManaged: React.FC<Props> = ({
...
   const cellarConfig = cellarData.config
   const { data: tvm } = useTvm(cellarConfig)
   const { data: apy, isLoading: apyLoading } = useApy(cellarConfig)
   const { data: cellarCap } = useCellarCap(cellarConfig)
   const { data: currentDeposits } = useCurrentDeposits(cellarConfig)
   const { data: activeAsset } = useActiveAsset(cellarConfig)
+  const { data: tokenPrice } = useTokenPrice(cellarConfig)
+  const { data: weekChange } = useWeekChange(cellarConfig)
+
+  const valueManagedData = {
```
[the remark this answers is not in the document]
Comment (codingki) [reply src/components/_cards/CellarCard/ValueManaged.tsx:44]: and branching in the UI with `uiConfig.ts` -> `isTVMEnabled` and `isTokenPriceEnabled

File: src/data/CellarHomeDataMap.ts [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import { config } from "utils/config"
+import { CellarDataMap } from "./types"
+import { aaveStable, ethBtcTrend, ethBtcMomentum } from "./strategies"
+
+interface CellarHomeDataMap {
+  automatedPortofolio: CellarDataMap
+  yieldStrategies: CellarDataMap
+}
+
+let cellarHomeConfig: CellarHomeDataMap
```
Comment (codingki) [inline src/data/CellarHomeDataMap.ts:10]: We don't want to have too many source data constant, let's move this to `cellarDataMap` and add a key `type` with enum `automatedPortfolio` and `yieldStrategies` there and in `PageHome` we show it with filtering the `type`.

File: src/data/CellarHomeDataMap.ts [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+cellarHomeConfig = {
+  automatedPortofolio: {
```
Comment (codingki) [inline src/data/CellarHomeDataMap.ts:13]: portfolio

**Review by codingki — CHANGES_REQUESTED** [review]

File: src/data/CellarHomeDataMap.ts (hunk 5 above)
Comment (joshuanatanielnm) [reply src/data/CellarHomeDataMap.ts:13]: thankssss

File: src/components/CellarStats.tsx (hunk 1 above)
Comment (joshuanatanielnm) [reply src/components/CellarStats.tsx:21]: do you have better suggestion? 
I used that because we don't use spesific value for that component

Comment (joshuanatanielnm) [reply src/components/CellarStats.tsx:21]: or maybe should I just create seperate component?

**Review by codingki — APPROVED** [review]
Approved as Henri requested, because this PR blocks marketing team

**Pull request merged**